tests/slack/test_slack_mentions.py

```diff
@@ -0,0 +1,20 @@
+from integrations.slack import SlackSender
+
+
+def test_title_without_mentions():
+    title = "Title with no mentions"
+    new_title, mention = SlackSender.extract_mentions(title)
+    assert new_title == title
+    assert mention == ""
+
+def test_title_with_single_mention():
+    title = "Title with single mention <@U024BE7LH>"
+    new_title, mention = SlackSender.extract_mentions(title)
+    assert new_title == "Title with single mention"
+    assert mention == " <@U024BE7LH>"
+
+def test_title_with_two_mentions():
+    title = "Title with <@U024BE7LG> two mentions <@U024BE7LH>"
+    new_title, mention = SlackSender.extract_mentions(title)
+    assert new_title == "Title with  two mentions"
```

Comment:
Are we fine with the double space in that case? Just double checking that you noticed it and ok with that.

Reply:
I think it's fine